charts/nextcloud/values.yaml

```diff
@@ -31,7 +31,9 @@ ingress:
   #    server_tokens off;
   #    proxy_hide_header X-Powered-By;
 
-  #    rewrite ^/.well-known/webfinger /public.php?service=webfinger last;
+  #    rewrite ^/.well-known/webfinger /index.php/.well-known/webfinger last;
+  #    rewrite ^/.well-known/nodeinfo /index.php/.well-known/nodeinfo last;
+
```

Comment:
Is there any reason we need this newline here?  It makes sense to group the rewrites together.